pom.xml

```diff
@@ -251,5 +251,10 @@
       <scope>compile</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.reflections</groupId>
+      <artifactId>reflections</artifactId>
```

Comment:
According to https://github.com/ronmamo/reflections#contribute, this is WTFPL licensed ... seeing http://www.wtfpl.net/about/, do you think that is not a problem for this project and more importantly to move this into EPL'd ODL infrautils?

Reply:
IANAL, but the WTFPL is fine, it’s considered free (*e.g.* [Fedora](https://fedoraproject.org/wiki/Licensing:Main#Good_Licenses), [Debian](https://wiki.debian.org/DFSGLicenses#DO_WHAT_THE_FUCK_YOU_WANT_TO_PUBLIC_LICENSE)) and is permissive and compatible with EPL (1 and 2).